src/main/java/org/dswarm/graph/json/deserializer/StatementDeserializer.java

```diff
@@ -17,26 +17,36 @@
 
 import java.io.IOException;
 
+import org.dswarm.graph.json.LiteralNode;
+import org.dswarm.graph.json.Node;
+import org.dswarm.graph.json.Predicate;
+import org.dswarm.graph.json.ResourceNode;
+import org.dswarm.graph.json.Statement;
+
 import com.fasterxml.jackson.core.JsonParseException;
 import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.ObjectCodec;
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JsonDeserializer;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.JsonNodeType;
 
-import org.dswarm.graph.json.LiteralNode;
-import org.dswarm.graph.json.Node;
-import org.dswarm.graph.json.Predicate;
-import org.dswarm.graph.json.ResourceNode;
-import org.dswarm.graph.json.Statement;
-
 /**
  * @author tgaengler
  */
 public class StatementDeserializer extends JsonDeserializer<Statement> {
 
+	private static final String ID = "id";
+	private static final String UUID = "uuid";
+	private static final String SUBJECT = "s";
+	private static final String URI = "uri";
+	private static final String PREDICATE = "p";
+	private static final String OBJECT = "o";
+	private static final String VALUE = "v";
+	private static final String ORDER = "order";
+	private static final String EVIDENCE = "evidence";
+	private static final String CONFIDENCE = "confidence";
+
```

Comment:
Not dswarm-gdm, later in this PR, I discovered https://github.com/zazi/dswarm-gdm-java/blob/json-streaming/src/main/java/org/dswarm/graph/json/stream/ModelStatics.java that already contains these identifiers, so we can use these.